README.md

```diff
@@ -15,57 +15,64 @@ products:
 
 ![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)
 
-These sample projects showcase how to build Unity applications 
+These sample projects showcase how to build Unity applications
 for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin.
 For more details on installing related tools and setting up a Unity project,
 please reference [the plugin documentation on https://docs.microsoft.com/](https://aka.ms/openxr-unity).
 
 > NOTE: This repository uses [Git Large File Storage](https://git-lfs.github.com/) to store large files,
-such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
+> such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
 
 > NOTE: Features using AR Raycasts and AR Planes require the latest preview OpenXR runtime,
-which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions).
-To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
+> which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions).
+> To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
 Recommend to run these samples on HoloLens 2 using following version of tools:
```

Comment:
Can you add a ## to make this a title that can be linked?  suggest:

** Recommend tools versions

Reply:
Added!